### PR TITLE
text to speech

### PR DESCRIPTION
**Pull request by emmyrunge**

### CODE REVIEW

File: src/main/java/touroll/tts/TtoS.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import javax.sound.sampled.AudioFileFormat;
+import javax.sound.sampled.AudioFormat;
+import javax.speech.AudioException;
+import javax.speech.Central;
+import javax.speech.EngineException;
+import javax.speech.synthesis.Synthesizer;
+import javax.speech.synthesis.SynthesizerModeDesc;
+import java.util.Locale;
+
+public class TtoS
```
Comment (andrewoid) [inline src/main/java/touroll/tts/TtoS.java]: Abbreviations are bad.

File: textToFile.wav
Comment (andrewoid) [inline textToFile.wav:1]: This file should not be included.

File: src/main/java/touroll/tts/TtoS.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        // Allocate synthesizer
+        synthesizer.allocate();
+
+        // Resume Synthesizer
+        synthesizer.resume();
+
+        SingleFileAudioPlayer singleFileAudioPlayer = new SingleFileAudioPlayer(
+                "textToFile", AudioFileFormat.Type.WAVE);
+        singleFileAudioPlayer.setAudioFormat(new AudioFormat(
+                44100, 16, 2, true, true));
```
Comment (andrewoid) [inline src/main/java/touroll/tts/TtoS.java]: Does this work?

**Comment by emmyrunge** [comment]
should I comment out or delete the checkForFile test since it doesn't work and my build will fail?

**Comment by andrewoid** [comment]
> should I comment out or delete the checkForFile test since it doesn't work and my build will fail?

Delete it.

File: src/test/java/touroll/tts/textToAudioTest.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import javax.sound.sampled.LineUnavailableException;
+import javax.speech.AudioException;
+import javax.speech.EngineException;
+import java.io.File;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class textToAudioTest
```
[the remark this answers is not in the document]
Comment (andrewoid) [reply src/test/java/touroll/tts/textToAudioTest.java]: You need to fix this.

File: src/main/java/touroll/tts/textToAudio.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import javax.sound.sampled.AudioFileFormat;
+import javax.sound.sampled.AudioFormat;
+import javax.speech.AudioException;
+import javax.speech.Central;
+import javax.speech.EngineException;
+import javax.speech.synthesis.Synthesizer;
+import javax.speech.synthesis.SynthesizerModeDesc;
+import java.util.Locale;
+
+public class textToAudio
```
[the remark this answers is not in the document]
Comment (andrewoid) [reply src/main/java/touroll/tts/textToAudio.java]: You need to fix this.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**